### PR TITLE
[Bulk] Validation - Missing descriptions

### PR DESCRIPTION
**Pull request by vatmicro**
This PR addresses 27 missing description issues.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by vatmicro** [comment]
Global effort to fix validation errors

@adegeo 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-automatic-layout-to-create-a-button.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Use Automatic Layout to Create a Button"
+description: Learn how to use Automatic layout to create a button in a localizable application in Windows Presentation Foundation (WPF).
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-automatic-layout-to-create-a-button.md:3]:
```suggestion
description: Learn how to use automatic layout to create a button in a localizable application in Windows Presentation Foundation (WPF).
```

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-system-fonts-keys.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Use System Fonts Keys"
+description: Learn how to use System Fonts keys, which is a system resource that helps you to create visuals that are consistent with system settings.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-system-fonts-keys.md:3]:
```suggestion
description: Learn how to use system font keys, which is a system resource that helps you to create visuals that are consistent with system settings.  
```

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-system-parameters-keys.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Use System Parameters Keys"
+description: Learn how to use System Parameters keys to expose system metrics that help you create visuals that are consistent with system settings.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-system-parameters-keys.md:3]:
```suggestion
description: Learn how to use system parameters keys to expose system metrics that help you create visuals that are consistent with system settings.
```

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-systemfonts.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Use SystemFonts"
+description: Learn how to use SystemFonts, which is a static resource belonging to the xref:System.Windows.SystemFonts class.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-systemfonts.md:3]:
```suggestion
description: An example of using the SystemFonts class to style and customize a button in Windows Presentation Foundation (WPF).
```

File: dotnet-desktop-guide/framework/wpf/advanced/how-to-use-systemparameters.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Use SystemParameters"
+description: Learn how to use the xref:System.Windows.SystemParameters class and its properties to style or customize a button.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/how-to-use-systemparameters.md:3]:
```suggestion
description: Learn how to use the SystemParameters class and its properties to style or customize a button.
```

File: dotnet-desktop-guide/framework/wpf/advanced/loadfromhistory-function-wpf-unmanaged-api-reference.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "LoadFromHistory Function - WPF unmanaged API reference"
+description: Learn about the LoadFromHistory function and how it works with Unmanaged API Reference in Windows Presentation Foundation (WPF).
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/loadfromhistory-function-wpf-unmanaged-api-reference.md:3]:
```suggestion
description: Learn about the LoadFromHistory function and how it works with unmanaged API Reference in Windows Presentation Foundation (WPF).
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**